uber/templates/regform.html

```diff
@@ -161,17 +161,17 @@
         </div>
         {% popup_link "../static_views/affiliates.html" "What's an affiliate?" %}
     </div>
-{% endif %}
 
```

Comment:
I originally left the shirt row outside of the if-block because we wanted it on the admin form.  However, we already had it separately on the admin form, which meant it was showing up twice.  This was causing problems on the admin form, since the shirt would be submitted twice, and the backend would complain if the size changed, since it would only be changed in one of the two places, and it would be parsed as a list rather than an integer, etc.

This change just moves it inside the if block so on the admin form it doesn't get generated here.